packages/css/src/components/document/document.scss

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@mixin reset-body {
+  margin-block: 0;
+  margin-inline: 0;
+}
+
+/**
+ * Add this class to your HTML `<body>` element.
+ * It overrides margins from the user agent styles,
+ * ans sets a white background and full height.
+ */
+.ams-body {
+  @include reset-body;
+
+  background-color: var(--ams-color-primary-white);
+  min-height: 100vh;
```

Comment:
Are these necessary? The background color is always white for me, and it isn't an issue if the document is shorter than the screen.

Reply:
I'd only add both if they're necessary, maybe we can make it a prop on Screen?